apidoc/Titanium/Network/HTTPClient.yml

```diff
@@ -233,6 +233,18 @@ methods:
         type: String
     platforms: [android, iphone, ipad, macos]
 
+  - name: getAllResponseHeaders
+    summary: All of the response headers.
+    description: |
+        Contains a single string, or an empty string if no headers are available.
+
+        See also: [getResponseHeader](Titanium.Network.HTTPClient.getResponseHeader).
+
+        Intended to match the `XMLHTTPRequest` [getAllResponseHeaders API](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/getAllResponseHeaders).
+    returns:
+        type: String
+    since: {iphone: "10.0.0", ipad: "10.0.0", macos: "10.0.0"}
+
```

Comment:
Android already has an `allResponseHeaders` property and `getAllResponseHeaders()` method. It looks like you didn't remove the getter for this on Android. So, you should document this for Android too. Note that Android has had this API for years.

FYI: This used to be a parity issue between Android and iOS. We recently added `responseHeaders` property support to Android in 9.3.0 to match iOS, which returns the headers as a dictionary.
https://jira.appcelerator.org/browse/TIMOB-24983

Reply:
We document that `allResponseHeaders` was added to Android as of v0.8.

Reply:
Ok, I'll add android since 0.8 here as well. I explicitly didn't remove this because it's intended to mirror the XMLHTTPRequest API from browsers (which is what HTTPClient is modeled on).